FFXIVClientStructs/FFXIV/Client/UI/NumberArray/PartyListNumberArray.cs

```diff
@@ -0,0 +1,72 @@
+namespace FFXIVClientStructs.FFXIV.Client.UI.NumberArray;
+
+[NumberArray(4)]
+[StructLayout(LayoutKind.Explicit, Size = 0x04 * 727)]
+public unsafe partial struct PartyListNumberArray {
+    public const int PartyMemberNumberCount = 42;
+
+    [FieldOffset(0x04 * 002)] public int PartyLeaderIndex;
```

Comment:
An alternative to doing FieldOffset for every property:
    [FieldOffset(0)] private fixed int IntArray[727];
    public int PartyLeaderIndex => this.IntArray[2];

Thoughts?

Reply:
I don't think this would work too well with structs inside the arrays